nautobot/extras/views.py

```diff
@@ -24,6 +24,7 @@
 from nautobot.dcim.models import Device
 from nautobot.dcim.tables import DeviceTable
 from nautobot.extras.utils import get_job_content_type, get_worker_count
+from nautobot.extras.tables import NestedDynamicGroupAncestorsTable, NestedDynamicGroupDescendantsTable
```

Comment:
We're already importing `.tables` below, could just refer to `tables.NestedDynamicGroup*Table` instead of importing them separately.